### PR TITLE
Tag LinearOperators.jl v0.3.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JuliaSmoothOptimizers/LinearOperators.jl](https://github.com/JuliaSmoothOptimizers/LinearOperators.jl)
Release: [v0.3.0](https://github.com/JuliaSmoothOptimizers/LinearOperators.jl/releases/tag/v0.3.0)
Diff: [vs v0.2.0](https://github.com/JuliaSmoothOptimizers/LinearOperators.jl/compare/c1d9adbd6cefbad82af7040df52ae47d371d6c5d...ab5c551d6df0bbbb67b7b5645a474773e5b0155f)
`requires` vs v0.2.0: 
```diff
--- 0.2.0/requires
+++ 0.3.0/requires
@@ -1 +1,2 @@
 julia 0.4
+Compat 0.20
```
cc: @abelsiqueira

### CODE REVIEW

**Comment by abelsiqueira** [comment]
@dpo

File: LinearOperators/versions/0.3.0/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.4
+Compat 0.20
```
Comment (tkelman) [inline LinearOperators/versions/0.3.0/requires]: the newest thing you're using is abstract type which was 0.17

Comment (abelsiqueira) [inline LinearOperators/versions/0.3.0/requires]: Is there an easy way to find the minimum required version of things of required packages?

Comment (tkelman) [inline LinearOperators/versions/0.3.0/requires]: for Compat, git blame on the readme usually works

Comment (abelsiqueira) [inline LinearOperators/versions/0.3.0/requires]: Thanks

**Pull request merged**